packages/suite/src/actions/wallet/sendFormActions.ts

```diff
@@ -250,7 +251,41 @@ const pushTransaction =
                 }),
             );
 
-            if (precomposedTx.prevTxid) {
+            const isRbf = precomposedTx.prevTxid !== undefined;
+
+            if (isRbf) {
+                const accountMetadata = selectLabelingDataForAccount(getState(), account.key);
+                const oldLabeling = accountMetadata?.outputLabels?.[precomposedTx.prevTxid];
+
+                const synchronize = getSynchronize();
+
+                Object.entries(oldLabeling).forEach(([outputIndex, value]) => {
+                    synchronize(() =>
+                        dispatch(
+                            metadataActions.addMetadata({
+                                type: 'outputLabel',
+                                entityKey: account.key,
+                                txid: precomposedTx.prevTxid, // delete original label
+                                outputIndex: Number(outputIndex),
+                                defaultValue: '',
+                                value: '',
+                            }),
+                        ),
+                    );
+                    synchronize(() =>
+                        dispatch(
+                            metadataActions.addMetadata({
```

Comment:
Maybe it would be good to handle labeling in the `replaceTransactionThunk` but this is insuite-common/wallet-core` and labling code is not available there.

Maybe it would be good to extract labeling from the suite and make it separate so it can be used in mobile one day as well.